.cg.yml

```diff
@@ -0,0 +1,37 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - main
+      - "refs/tags/ccf-*"
+      - cwinter_com_gov
```

Comment:
Presumably to be removed before merging?

Reply:
Yep, thanks!